docs/data/data-grid/recipes/recipes.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Data Grid - Recipes
+---
+
+# Data grid - Recipes
+
+<p class="description">Recipes for grid customization.</p>
+
+## Editing
+
+### Multiline editing
+
+Creating custom component for inline editing of multiline text.
```

Comment:
Not sure about the form - should it be more like a tutorial with step-by-step explanation or just a working demo that users can explore on their own?

Reply:
It depends. This specific demo uses only the principles already explained in https://deploy-preview-5744--material-ui-x.netlify.app/x/react-data-grid/editing/#create-your-own-edit-component. Its "advanced" part is more of an advanced usage of MUI components.

Reply:
For this specific demo I do not see how to split the explanation. It could contain one or two sentences about the big lines. Especially on the useof `'cellKeyDown'` which is not intuitive..

The strategy is to use a custom editing component that renders a `Popper` over the cell. In this proper you have enough space to use a multiline input.
Since the component is in the Popper, keyboard event will not be propagated to the grid. Then you need to publish the 'cellKeyDown' such that keyboard interaction like Ctrl+Enter or Escape quite the edit mode and saves or reverts the editing.

Reply:
Prefer active voice.

```suggestion
You can have columns with multiline text and edit them by creating a custom edit component.
```